### PR TITLE
Doc Fix: added note about cuda9.2 requirement to Java example

### PR DESCRIPTION
**Pull request by samskalicky**
## Description ##
example requires CUDA 9.2 but this wasnt mentioned anywhere in the instructions

### Changes ###
Added note to clarify the CUDA 9.2 requirement

### CODE REVIEW

File: scala-package/mxnet-demo/java-demo/README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ This command will pick the default values specified in the [pom](https://github.
 
 Note: If you are planning to use GPU, please add `-Dmxnet.profile=linux-x86_64-gpu`
 
+Note: The Maven package is built with CUDA 9.2.
```
Comment (piyushghai) [inline scala-package/mxnet-demo/java-demo/README.md]: Thanks Sam for bringing this up.
I think this is the correct location for this disclaimer : https://github.com/apache/incubator-mxnet/tree/master/scala-package#pre-built-maven-packages

Comment (samskalicky) [reply scala-package/mxnet-demo/java-demo/README.md]: hmmm... I never looked at this page before. Maybe we should add a link and generalize the note more and say something like "Make sure you're environment is configured correctly according to..." and link to that page then?

Users following this demo arent necessarily going to read all the docs before running. We need to keep things self-containted and succinct.

**Comment by ankkhedia** [comment]
@samskalicky Thanks for your contribution!

@mxnet-label-bot add [pr-awaiting-review]

**Comment by anirudhacharya** [comment]
@zachgk will the coverage results get uploaded to the codecov report that is generated for all of mxnet on CI runs?

**Comment by karan6181** [comment]
@samskalicky Can you please address the review comments ? Thanks!

**Comment by abhinavs95** [comment]
@mxnet-label-bot update [pr-awaiting-response]

**Comment by piyushghai** [comment]
@lanking520 @samskalicky @zachgk Can we take this PR to completion ?

**Pull request merged**